packages/preview/orange-book/0.1.0/README.md

```diff
@@ -0,0 +1,70 @@
+# orange-book
+A book template inspired by The Legrand Orange Book of Mathias Legrand and Vel https://www.latextemplates.com/template/legrand-orange-book.
+
+## Usage
+You can use this template in the Typst web app by clicking "Start from template"
+on the dashboard and searching for `orange-book`.
+
+Alternatively, you can use the CLI to kick this project off using the command
+```
+typst init @preview/orange-book
```

Comment:
Looks like this doesn't work - however adding the version does:

```
typst init @preview/orange-book:0.1.0
```

Reply:
I think the reason is that package index is only re-built once a day, so you may have to wait up to 24h before the compiler is aware that the 0.1.0 is the latest available.